### PR TITLE
[REM] project: remove 'Tasks by deadline' indication from calendar view

### PR DESCRIPTION
**Pull request by deneuvillem**
Since the 'end date' and the 'deadline' have been merged, there should no longer be any confusion as to what the calendar view represents.

task-3696660

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/static/tests/views/project_task_calendar/project_task_calendar_view_tests.js [hunk 1]
```diff
@@ -48,6 +48,6 @@ QUnit.module("Project Task Calendar View", (hooks) => {
         };
         const webClient = await createWebClient({ serverData });
         await doAction(webClient, 1);
-        assert.equal(document.querySelector(".o_last_breadcrumb_item span").textContent, 'test - Tasks by Deadline');
+        assert.equal(document.querySelector(".o_last_breadcrumb_item span").textContent, 'test');
     });
```
Comment (MissingNoShiny) [inline addons/project/static/tests/views/project_task_calendar/project_task_calendar_view_tests.js:52]: I think you can remove the test, it's not useful anymore

File: addons/project/static/src/views/project_task_calendar/project_task_calendar_controller.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export class ProjectTaskCalendarController extends CalendarController {
     };
     setup() {
         super.setup(...arguments);
-        this.env.config.setDisplayName(this.env.config.getDisplayName() + _t(" - Tasks by Deadline"));
+        this.env.config.setDisplayName(this.env.config.getDisplayName());
     }
```
Comment (MissingNoShiny) [inline addons/project/static/src/views/project_task_calendar/project_task_calendar_controller.js:16]: You can remove the whole setup, the line doesn't do anything anymore

**Review by MissingNoShiny — COMMENTED** [review]
Thanks for your work, I left two comments :slightly_smiling_face:

**Review by xavierbol — APPROVED** [review]
robodoo r+

**Pull request closed without merging**